CoolestMovieAPI/CoolestMovieAPI/Services/MovieRepository.cs

```diff
@@ -16,43 +16,46 @@ namespace CoolestMovieAPI.Services
     public class MovieRepository : BaseRepository, IMovieRepository
     {       
         public MovieRepository(MovieContext movieContext, ILogger<MovieRepository> logger) : base (movieContext, logger)
-        {
-                                                    
-        }
+        {}
         
        public async Task<IList<Movie>> GetAllMovies()          
        {
-            return await _movieContext.Movies.Where(_ => true).ToListAsync();                               
+            _logger.LogInformation($"Getting all movies!");
+            
+            var query = await _movieContext.Movies
+                .Where(_ => true)
```

Comment:
Is this Where necessary to get all movies?

Reply:
No it's not, removed in next commit